xmipptomo/protocols/protocol_subtraction_subtomo.py

```diff
@@ -77,87 +75,29 @@ def _defineParams(self, form):
                       expertLevel=LEVEL_ADVANCED, help='Save input volume 1 (first subtomogram of the set) filtered '
                                                        'and input volume 2 (average) adjusted, which are the volumes '
                                                        'that are really subtracted.')
+        form.addParallelSection(threads=0, mpi=4)
```

Comment:
Why 4 and not 8? (is the default in other cases)

Reply:
Why we can not fill the MPI number in the form?
![image](https://github.com/I2PC/scipion-em-xmipptomo/assets/12760268/f16fcd50-eec4-4bcc-a7b4-db26d7389546)

Reply:
Despite I am still learning the algorithm the MPI can overload the RAM. That means at least 8 copies in memory, but the algorithm can requires many more (internaly)

Reply:
Yes, and AFAIK 4 was the maximum allowed in the machine where Scipion tests were executed (maybe this has changed now, not sure). But I have no problem if you change it as you consider.

With respect to why we can not fill the MPI number in the form, I could when I tried (actually I tried with different values), so not sure what has changed (maybe something from scipion is not updated or has changed?). I am so sorry but I can not check it now as my Ubuntu has died (I do not know why... the destiny?) but let me know if you need something @albertmena .